topdown/strings.go

```diff
@@ -10,10 +10,180 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/tchap/go-patricia/v2/patricia"
+
 	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/topdown/builtins"
 )
 
+func builtinAnyPrefixMatchesAnyString(a, b ast.Value) (ast.Value, error) {
+
+	var strs []string
+	switch a := a.(type) {
+	case *ast.Array:
+		err := a.Iter(func(x *ast.Term) error {
+			s, ok := x.Value.(ast.String)
+			if !ok {
+				return builtins.NewOperandElementErr(2, b, x.Value, "string")
```

Comment:
The operand number seems to be set to `2` for all errors here..

Reply:
fixed